pkg/node/types/node.go

```diff
@@ -151,6 +204,16 @@ func (n *RegisterNode) DeepKeyCopy() store.LocalKey {
 	return n.DeepCopy()
 }
 
+// Interface contains details about an interface.
+//
+// +k8s:deepcopy-gen=true
+type Interface struct {
+	Gateway           Address
```

Comment:
Why does this need to be of an `Address` type? [1]

Reply:
This is no longer needed.